lib/index.js

```diff
@@ -58,13 +58,23 @@ module.exports = {
           }
         }
 
+        const executable = (
+          this.executablePath ||
+          await findCachedAsync(fileDirectory, 'node_modules/.bin/flow') ||
```

Comment:
I feel like it is, since I changed the default to `""` the order of importance is the `executablePath` user setting,  the discovered local `flow` executable, and finally the global `flow` install.

IMO if you set an executable path in the user settings flow-ide shouldn't use anything else.